xml/System.Data.Objects.DataClasses/EntityCollection`1.xml

```diff
@@ -463,7 +463,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.ValidateLoad%2A> method before loading the collection. See this method for a list of possible exceptions.  
+ This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.Load*> method before loading the collection. See this method for a list of possible exceptions.  
```

Comment:
I can't do anything that requires too much brain late at night. Yes, your replacement seems fine.